diary_emotion.py

```
@@ -1,5 +1,5 @@
 from azure.ai.projects import AIProjectClient
-from azure.identity import DefaultAzureCredential, ClientSecretCredential, ManagedIdentityCredential
+from azure.identity import DefaultAzureCredential, ClientSecretCredential
```

Pylint (3.3.7)

[error] Unable to import 'azure.identity'

(E0401)